src/libraries/apicompat/ApiCompatBaselineExcludedAttributes.txt

```diff
@@ -0,0 +1,56 @@
+T:System.Configuration.ConfigurationPropertyAttribute
+T:System.Diagnostics.CodeAnalysis.DynamicDependencyAttribute
+T:System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute
+T:System.Diagnostics.CodeAnalysis.MemberNotNullAttribute
+T:System.Diagnostics.CodeAnalysis.SuppressMessageAttribute
+T:System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute
+T:System.Diagnostics.DebuggerBrowsableAttribute
+T:System.Diagnostics.DebuggerDisplayAttribute
+T:System.Diagnostics.DebuggerHiddenAttribute
+T:System.Diagnostics.DebuggerNonUserCodeAttribute
+T:System.Diagnostics.DebuggerStepThroughAttribute
+T:System.Diagnostics.DebuggerTypeProxyAttribute
+T:System.Diagnostics.MonitoringDescriptionAttribute
+T:System.Diagnostics.StackTraceHiddenAttribute
+T:System.IO.IODescriptionAttribute
+T:System.Runtime.CompilerServices.AsyncIteratorStateMachineAttribute
+T:System.Runtime.CompilerServices.AsyncStateMachineAttribute
+T:System.Runtime.CompilerServices.CompilerGeneratedAttribute
+T:System.Runtime.CompilerServices.EnumeratorCancellationAttribute
+T:System.Runtime.CompilerServices.IteratorStateMachineAttribute
+T:System.Runtime.CompilerServices.MethodImpl
+T:System.Runtime.CompilerServices.NullableAttribute
+T:System.Runtime.CompilerServices.NullableContextAttribute
+T:System.Runtime.CompilerServices.TypeForwardedFromAttribute
+T:System.Runtime.ConstrainedExecution.PrePrepareMethodAttribute
+T:System.Runtime.ConstrainedExecution.ReliabilityContractAttribute
+T:System.Runtime.InteropServices.ClassInterfaceAttribute
+T:System.Runtime.InteropServices.ComDefaultInterfaceAttribute
+T:System.Runtime.InteropServices.ComVisibleAttribute
+T:System.Runtime.InteropServices.GuidAttribute
+T:System.Runtime.InteropServices.LCIDConversionAttribute
+T:System.Runtime.InteropServices.StructLayoutAttribute
+T:System.Security.Permissions.EnvironmentPermissionAttribute
+T:System.Security.Permissions.FileIOPermissionAttribute
+T:System.Security.Permissions.HostProtectionAttribute
+T:System.Security.Permissions.IsolatedStorageFilePermissionAttribute
+T:System.Security.Permissions.PermissionSetAttribute
+T:System.Security.Permissions.ReflectionPermissionAttribute
+T:System.Security.Permissions.SecurityPermissionAttribute
+T:System.Security.Permissions.StrongNameIdentityPermissionAttribute
+T:System.Security.SecurityCriticalAttribute
+T:System.Security.SecuritySafeCriticalAttribute
+T:System.Security.SuppressUnmanagedCodeSecurityAttribute
+T:System.Runtime.Versioning.NonVersionableAttribute
+T:System.Runtime.CompilerServices.IntrinsicAttribute
+T:System.Runtime.CompilerServices.ExtensionAttribute
+T:System.Drawing.SRDescriptionAttribute
+T:System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute
+T:System.Runtime.CompilerServices.AsyncMethodBuilderAttribute
+T:System.Reflection.DefaultMemberAttribute
+T:System.Timers.TimersDescriptionAttribute
+T:System.ComponentModel.EditorBrowsableAttribute
+T:System.ObsoleteAttribute
```

Comment:
This doesn't make sense to me. Why should `ObsoleteAttribute` be excluded from reference assemblies?

Reply:
This file lists the attributes that apicompat ignores when comparing the NetCoreAppCurrent reference assemblies against the net6.0 and netstandard2.0 and netstandard2.1 reference assemblies.